graphrag_eval/steps/evaluation.py

```diff
@@ -1,41 +1,49 @@
 import json
```

Comment:
Should we add a blank \_\_init__ file to the steps folder to make it a module?

Reply:
Added `__init__.py` that exposes `get_steps_evaluation_result_dict()`. Do you prefer it to be blank?

Reply:
I think it's better to be blank. I don't see what's the logic behind exposing get_steps_evaluation_result_dict, but not the others